When a valid credentials.xml exist in user home, the application crash at startup

java.lang.NoClassDefFoundError: com/mucommander/commons/io/base64/Base64Decoder
 at com.mucommander.bookmark.XORCipher.decryptXORBase64(XORCipher.java:103)
 at com.mucommander.auth.CredentialsParser.endElement(CredentialsParser.java:143)
 at java.xml/com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.endElement(AbstractSAXParser.java:609)
 at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEndElement(XMLDocumentFragmentScannerImpl.java:1713)
 at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:2879)
 at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:601)
 at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:531)
 at java.xml/com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:887)
 at java.xml/com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:823)
 at java.xml/com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:141)
 at java.xml/com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1213)
 at java.xml/com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:639)
 at java.xml/com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl.parse(SAXParserImpl.java:323)
 at java.xml/javax.xml.parsers.SAXParser.parse(SAXParser.java:196)
 at com.mucommander.auth.CredentialsParser.parse(CredentialsParser.java:78)
 at com.mucommander.auth.CredentialsManager.loadCredentials(CredentialsManager.java:169)
 at com.mucommander.muCommander.run(muCommander.java:459)
 at com.mucommander.muCommander.run(muCommander.java:582)
 at com.mucommander.Activator.start(Activator.java:54)
 at org.apache.felix.framework.util.SecureAction.startActivator(SecureAction.java:697)
 at org.apache.felix.framework.Felix.activateBundle(Felix.java:2239)
 at org.apache.felix.framework.Felix.startBundle(Felix.java:2145)
 at org.apache.felix.framework.Felix.setActiveStartLevel(Felix.java:1372)
 at org.apache.felix.framework.FrameworkStartLevelImpl.run(FrameworkStartLevelImpl.java:308)
 at java.base/java.lang.Thread.run(Thread.java:844)
Caused by: java.lang.ClassNotFoundException: com.mucommander.commons.io.base64.Base64Decoder not found by org.mucommander.core [37]
 at org.apache.felix.framework.BundleWiringImpl.findClassOrResourceByDelegation(BundleWiringImpl.java:1610)
 at org.apache.felix.framework.BundleWiringImpl.access$200(BundleWiringImpl.java:80)
 at org.apache.felix.framework.BundleWiringImpl$BundleClassLoader.loadClass(BundleWiringImpl.java:2018)
 at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:496)
 ... 25 more

--- a/mucommander-commons-io/build.gradle
+++ b/mucommander-commons-io/build.gradle
@@ -30,6 +30,7 @@ jar {
          'org.slf4j'
         instruction 'Export-Package',
          'com.mucommander.commons.io',
+         'com.mucommander.commons.io.base64',
          'com.mucommander.commons.io.bom',
          'com.mucommander.commons.io.security'
        }

--- a/mucommander-core/build.gradle
+++ b/mucommander-core/build.gradle
@@ -60,6 +60,7 @@ jar {
         'com.mucommander.commons.file.protocol.local',
         'com.mucommander.commons.file.util',
         'com.mucommander.commons.io',
+        'com.mucommander.commons.io.base64',
         'com.mucommander.commons.io.bom',
         'com.mucommander.commons.io.security',
         'com.mucommander.commons.util',

